Cargo.toml

```diff
@@ -40,3 +44,48 @@ members = [
   "runtime-raw",
   "runtime-tokio",
 ]
+
+[[example]]
+name = "guessing"
+path = "examples/guessing.rs"
+required-features = ["native"]
+
+[[example]]
+name = "hello"
+path = "examples/hello.rs"
+required-features = ["native"]
+
+[[example]]
+name = "tcp-client"
+path = "examples/tcp-client.rs"
+required-features = ["native"]
+
+[[example]]
+name = "tcp-echo"
+path = "examples/tcp-echo.rs"
+required-features = ["native"]
+
+[[example]]
+name = "tcp-proxy"
+path = "examples/tcp-proxy.rs"
+required-features = ["native"]
+
+[[example]]
+name = "udp-client"
+path = "examples/udp-client.rs"
+required-features = ["native"]
+
+[[example]]
+name = "udp-echo"
+path = "examples/udp-echo.rs"
+required-features = ["native"]
+
+[[test]]
+name = "native"
+path = "tests/native.rs"
+required-features = ["native"]
+
+[[bench]]
+name = "native"
+path = "benches/native.rs"
+required-features = ["native"]
```

Comment:
I'm not too familiar with how default features work; don't the examples, tests, and benches work without explicitly setting that they need to work with the `native` feature?

Reply:
Of course, the normal tests will work without the explicit setting.

But, `cargo test --no-default-features` will not work. The explicit setting makes those tests ignored when the test is run without the `native` feature.

Reply:
Oh, I think we can remove this now too, right?

Reply:
I removed it and squash into one commit (because it's now super simple PR)